packages/platform-sdk-ada/src/services/client.test.ts

```diff
@@ -19,14 +19,18 @@ beforeAll(() => nock.disableNetConnect());
 describe("ClientService", function () {
 	it("#wallet should succeed", async () => {
 		nock(/.+/)
-			.get("/v2/wallets/98c83431e94407bc0889e09953461fe5cecfdf18")
-			.reply(200, require(`${__dirname}/../../test/fixtures/client/wallet.json`));
+			.post("/")
+			.reply(200, require(`${__dirname}/../../test/fixtures/client/transactions-0.json`))
+			.post("/")
+			.reply(200, require(`${__dirname}/../../test/fixtures/client/transactions-20.json`))
+			.post("/")
+			.reply(200, require(`${__dirname}/../../test/fixtures/client/utxos-aggregate.json`));
 
-		const result = await subject.wallet("98c83431e94407bc0889e09953461fe5cecfdf18");
```

Comment:
@faustbrian We probably need to define what we consider a wallet id. `98c83431e94407bc0889e09953461fe5cecfdf18` was the one that cardano-wallet returns that we couldn't figure out how to calculate.
So I changed this for using the public key of the account (not sure it's the best idea, but I had to choose something).

Reply:
Can we use the address? That's what we usually pass in to this method.

Reply:
Which address? There's no such thing as an address (that I know of)

Reply:
I'll make some changes tomorrow to abstract the knowledge about the specific input.

Reply:
https://docs.cardano.org/projects/cardano-wallet/en/latest/About-Address-Derivation.html#hd-sequential-wallets-a-la-bip-44

Reply:
Ok. Meanwhile I'll do the reminder (transactions, which should be a subcase of what I already did).

Reply:
Will have to think a bit about how to abstract away the specifics about the input because we allow people to import their wallets by an address in the desktop wallet, which won't be possible in this case.

Reply:
My guess is for Cardano users, the wallet id provided by `cardano-wallet` could be something they're used to. Perhaps not for end users because, at least in Daedalus, it's never shown. They just import a passphrase and the wallets are imported.